evalml/tests/component_tests/test_utils.py

```diff
@@ -115,10 +115,7 @@
     ]
 )
 not_supported_in_linux_py39 = set(
-    [
-        "ARIMA Regressor",
-        "Polynomial Detrender",
-    ]
+    ["ARIMA Regressor", "Polynomial Detrender", "Exponential Smoothing Regressor"]
```

Comment:
Hmmm, just curious on your thoughts @freddyaboulton: is there something we could have done to better catch this without using the nightlies? I guess not since we don't run 3.9 on each commit...

Reply:
I am happy that the error was pretty clear in the failure though: https://github.com/alteryx/evalml/runs/4728893144

Reply:
Yea maybe we should have expected the nightlies/conda to fail when we add an sktime component but ultimately I think it's ok that this was caught after merge during the nightlies. We decided to only test 3.9 during the nightlies to save on developer time for 95% of prs that don't exhibit any behavior between python < 3.9 and python 3.9 so fixing stuff is just part of the trade-off.